Utilities/NPC.gd

```diff
@@ -101,15 +102,16 @@ func animate(_dir):
 			"Right":
 				$AnimationPlayer.play("Right2")
 
-func set_idle_frame(_dir):
-	match _dir:
-		"Down":
+func set_idle_frame(idle):
+	facing = idle
+	match facing:
+		0:
```

Comment:
It's better to use the enum variables instead of it's values, to make the `match` clearer *(and to prevent bugs from creeping in)*.

*(e.g. change `0` to `directions.down`, and etcetera)*